CONTRIBUTING.md

```diff
@@ -70,11 +70,12 @@ commit automatically with `git commit -s`.
 ### Development
 
 - Create a personal fork of the project on GitHub
-- Use **sycl** branch as baseline for your changes
+  - For DPC++ Compiler project use **sycl** branch as baseline for your changes
 - Prepare your patch (follow
   [LLVM coding standards](https://llvm.org/docs/CodingStandards.html))
-- Build the project and run all tests (see
-[GetStartedWithSYCLCompiler.md](sycl/doc/GetStartedWithSYCLCompiler.md))
+- Build the project and run all tests
+  - For DPC++ Compiler project (see
```

Comment:
Not clear what is the value of adding a subitem in the middle.

Reply:
There could be more projects hosted in the future at intel/llvm, that's why all these subitems are there. I updated text a bit.